### ISSUE TITLE
publish .crx file for every release

### ISSUE THREAD

**Issue by lollilol** [austencm/youtube-auto-like#128]
Another extension that isn't available through chromes web store is "Bypass Paywalls Clean"

On the [releases page](https://gitlab.com/magnolia1234/bypass-paywalls-chrome-clean/-/releases), you can see that they upload a .crx file for every release. 

It would be a cool addition if you provided us with a .crx file, because it provides automatic updates and only requires you to add the extension id to an [allowlist](https://gitlab.com/magnolia1234/bypass-paywalls-chrome-clean/-/tree/master/allowlist/README.md)

**Comment by austencm** [comment]
Interesting, I'll definitely look into this!

**Comment by bremaya** [comment]
Yes! I use "Bypass Paywalls Clean" and the .crx files and updates work like a charm!

**Comment by austencm** [comment]
Still figuring this one out, but in the meantime if you're comfortable with git you can now update by loading the extension from the `/[dist](https://github.com/austencm/youtube-auto-like/tree/v.2.8.0/dist)` directory and pulling.

**Comment by bremaya** [comment]
> Another extension that isn't available through chromes web store is "Bypass Paywalls Clean"
> 
> On the [releases page](https://gitlab.com/magnolia1234/bypass-paywalls-chrome-clean/-/releases), you can see that they upload a .crx file for every release.
> 
> It would be a cool addition if you provided us with a .crx file, because it provides automatic updates and only requires you to add the extension id to an [allowlist](https://gitlab.com/magnolia1234/bypass-paywalls-chrome-clean/-/tree/master/allowlist/README.md)

Check this:
[https://github.com/bremaya/youtube-auto-like](url)
